settings/common.py

```diff
@@ -83,7 +83,7 @@
 
 ROOT_URLCONF = 'evalai.urls'
 
-TEMPLATE_PATH = os.path.join(BASE_DIR, 'frontend')
+TEMPLATE_PATH = os.path.join(BASE_DIR, 'frontend/dist')
```

Comment:
Can you please rewrite this as `TEMPLATE_PATH = os.path.join(BASE_DIR, 'frontend', 'dist')`